src/encoded/docs/schema-changes.md

```diff
@@ -211,11 +281,18 @@ There are 2 categories when we talk about updating an existing schema:
 
 
         def test_{metadata_object}_lowercase_property_1(app, {metadata_object}_2):
-            migrator = app.registry['migrator']
-            value = migrator.upgrade('{metadata_object}', {metadata_object}_2, target_version='3')
+            value = upgrader.upgrade('{metadata_object}', {metadata_object}_2, target_version='3')
             assert value['schema_version'] == '3'
             assert value['property_1'] == 'value 1'
-            
+
+**Specific example from the genetic modifications object upgrade**
+
+        def test_genetic_modification_upgrade_6_7(upgrader, genetic_modification_6):
+            value = upgrader.upgrade('genetic_modification', genetic_modification_6,
+                             current_version='6', target_version='7')
+            assert value['schema_version'] == '7'
+            assert value.get('purpose') == 'characterization'
+
 5. You must check the results of your upgrade on the current database:
    
    **note** it is possible to write a "bad" upgrade that does not prevent your objects from loading or being shown.
```

Comment:
It would be useful to add the printout of "batchupgrade" from a real demo - it will make it easy to understand what to look for